Jenkinsfile-bundle

```diff
@@ -0,0 +1,64 @@
+library("tdr-jenkinslib")
+
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    text(name: "BUILD_IMAGE_TAG", defaultValue: "latest", description: "The image tag for the base yara images")
+  }
+  stages {
+    stage("Build zip file") {
+      steps {
+        script {
+          sh "aws ecr get-login --region eu-west-2 --no-include-email | bash"
+          sh "docker stop dependencies | true"
+          sh "docker stop rules | true"
+          sh "rm -rf lambda"
```

Comment:
Is this to clean up the output of a previous job? I'm wondering if it would be safer to set the Jenkins job to do a clean checkout each time?

Reply:
Yep, there's a setting to delete all untracked files when the job runs, I've set this in the job settings and deleted the line to rm -rf lambda.